mind_matter_api/api/surveys.py

```
@@ -26,124 +28,198 @@ def get_surveys():
+        if isinstance(user_id, tuple): return user_id
+
         survey = svc.get_by_id(survey_id)
         if not survey:
             return jsonify({'error': 'Survey not found'}), 404
         return jsonify(SurveySchema().dump(survey)), 200
 
     @app.route('/surveys', methods=['POST'])
     def create_survey():
-        """Create a new survey"""
+        user_id = get_authenticated_user_id_or_abort()
+        if isinstance(user_id, tuple): return user_id
```

Ruff (0.8.2)

Multiple statements on one line (colon)

(E701)